src/Http/Controllers/StaticPagesController.php

```diff
@@ -2,33 +2,123 @@
 
 namespace Ourgarage\StaticPages\Http\Controllers;
 
+use Ourgarage\StaticPages\Http\Requests\StaticPageCreateRequest;
 use App\Http\Controllers\Controller;
+use Ourgarage\StaticPages\Models\StaticPage;
+use Notifications;
 
 class StaticPagesController extends Controller
 {
 
-    public function pageList() {
+    public function indexAdmin(StaticPage $staticPages)
+    {
+        $pages = $staticPages->orderBy('updated_at', 'desc')->paginate(20);
 
+        \Title::prepend(trans('dashboard.title.prepend'));
+        \Title::append(trans('static-pages::pages.index.title'));
+
+        if (view()->exists('packages.static-pages.admin.index')) {
+            return view('packages.static-pages.admin.index', ['pages' => $pages]);
```

Comment:
Not necessary to publish admin views. We need to publish only frontend part, because it can be stylized.
Admin should be untouched.